fix CHANGELOG.md (#1229)
lambdaclass/cairo-rs: CHANGELOG.md

#### Upcoming Changes

> * move the vm in it's own directory and crate, different from the workspace
> 
> * add a `ensure_no_std` crate that will be used by the CI to check that new changes are not reverting `no_std` support

Reviewer: ```suggestion
* move the vm to its own directory and crate, different from the workspace

* add an `ensure_no_std` crate that the CI will use to check that new changes don't revert `no_std` support
```
